tests/config/state/tests.yml

```diff
@@ -5,10 +5,11 @@ components:
     allOperations: true
   - component: mongodb
     allOperations: true
-  - component: cosmosdb
+  - component: azure.cosmosdb
     allOperations: true
   - component: sqlserver
     allOperations: true
   - component: mysql
-    allOperations: false
     operations: [ "set", "get", "delete", "bulkset", "bulkdelete", "transaction", "etag" ]
+  - component: azure.tablestorage
+    operations: ["set", "get", "delete", "etag", "bulkset", "bulkdelete"]
```

Comment:
Is `first-write` not supported by azure.tablestorage or is it that just it fails the first-write tests? Unlike `transaction` there's not a separate function implementation necessary for that test. 

(It looks like mysql may have the same issue, where the test was dropped because it could not pass?)

Reply:
Forgot to add this one. I fixed the first-write behavior now.